app/src/main/java/com/vero/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -0,0 +1,9 @@
+package com.vero.tasky.agenda.domain.model
+
+interface AgendaItem {
```

Comment:
An interface is usually used when you want the child to override methods

For overriding parameters, I personally recommend using a sealed class

Ex:
```
sealed class AgendaItem(
    val id: String,
    val title: String,
    val description: String,
    val time: LocalDateTime,
    val remindAt: LocalDateTime
) {
    data class Reminder(
    ...
    data class Task(
    ...
    data class Event(
    ...
}
```